platform/ios/INSTALL.md

```diff
@@ -95,8 +95,13 @@ If your application targets iOS 7.x, you’ll need to install the static framewo
 
 1. Build from source manually per above.
 
-1. Open the project editor and select your application target. Drag `build/ios/pkg/static/Mapbox.framework` into the “Embedded Binaries” section of the General tab. (Don’t drag it into the “Linked Frameworks and Libraries” section; Xcode will add it there automatically.) In the sheet that appears, make sure “Copy items if needed” is checked, then click Finish.
-
+1. Copy the contents of `build/ios/pkg/static` into your project. It should happen automatically, but ensure that:
```

Comment:
Developers have indicated that they’d like this step to be as explicit as possible, so let’s say to drag those files into the Project navigator, checking “Copy items if needed”.